### DISCUSSION TITLE
Déployez un serveur de messagerie en 2 mins !

### DISCUSSION THREAD

**Discussion by giscus** [zero-filtre/zerofiltre-blog#179] [Announcements]
# Déployez un serveur de messagerie en 2 mins !

Un serveur de messagerie basique mais fonctionnel + des conseils pour l'administrer et le protéger!

https://zerofiltre.tech/articles/186

**Comment by nexus9111** [comment]
Super article, pourrait tu montrer comment configurer le nom de domaine ?

**Reply by geek3000** [reply]
Hello

La configuration du nom de domaine se fait via les enregistrements DNS de type MX. Vous pouvez vous inspirer de cet article sur [hostinger ](https://support.hostinger.com/fr/articles/5670904-comment-configurer-un-nom-de-domaine-pour-l-e-mail-de-cpanel).

Bien sur le processus peut changer en fonction de votre hébergeur.

De plus, pendant la configuration n'oubliez pas les [bonnes pratiques](https://docker-mailserver.github.io/docker-mailserver/latest/config/best-practices/dkim_dmarc_spf/) qui permettrons à votre serveur d'être crédible

**Comment by padcmoi** [comment]
Excellente image, rapide et fonctionnelle, je vois dans le .env pour SASLAUTHD_MECHANISMS qu'il est possible de mettre MySQL en valeur mais rien d'autres
à quoi sert cette valeur ? 
peut on configurer les comptes dans une base de données MySQL ?
Cordialement.

**Reply by geek3000** [reply]
Hello désolé pour le retard.

Malheureusement, docker-mailserver ne permet pas la création de compte dans une base de donnée comme MySQL. Je vous recommande plutôt [mailu](https://mailu.io/2.0/database.html) qui est plus flexible.

Concernant l'authentification SASLAUTHD, je vous invite à consulter la [documentation](https://docker-mailserver.github.io/docker-mailserver/latest/config/environment/#saslauthd_mechanisms). Il prend en charge divers mécanismes d'authentification comme LDAP, RIMAP, etc
